packages/admin-ui/src/filters/use-filter-status.ts

```diff
@@ -0,0 +1,57 @@
+import { useState, useMemo } from 'react'
+
+import type { ComboboxState } from '../combobox/combobox.state'
+
+import type { AnyObject } from '@vtex/admin-ui-util'
+import type { FilterStatus } from './filter/filter.state'
+
+export function useFilterStatus<T extends AnyObject>(props: ComboboxState<T>) {
```

Comment:
The only difference between combobox status/setStatus and the new one created here are the status ids (loading-error, search-error, etc.)? If I get it right, IMO it would be better to have the same ids here and only reuse the combobox state

Reply:
No, there are two loadings, global loading and search loading. And so on

Reply:
Is this hook still useful since we have the same status?